i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md

```diff
@@ -5,39 +5,120 @@ title: Centreon SQL Metrics
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
```

Comment:
old text with link to the watch?
or clearer explanations as to what the connector actually does?